source/logHandler.py

```diff
@@ -350,14 +355,15 @@ def initialize(shouldDoRemoteLogging=False):
 def setLogLevelFromConfig():
 	"""Set the log level based on the current configuration.
 	"""
-	if globalVars.appArgs.debugLogging or globalVars.appArgs.logLevel != 0 or globalVars.appArgs.secure:
+	if globalVars.appArgs.debugLogging or globalVars.appArgs.logLevel != 0 or globalVars.appArgs.secure or globalVars.appArgs.noLogging:
 		# Log level was overridden on the command line or we're running in secure mode,
 		# so don't set it.
 		return
 	import config
 	levelName=config.conf["general"]["loggingLevel"]
 	level = levelNames.get(levelName)
-	if not level or level > log.INFO:
+	# The lone exception to level higher than INFO is "OFF" (100).

```

Comment:
I'd love there to be a "why" here. Levels between info and off are invalid? I'm going to guess it's because we only expose 
`OFF=100, INFO=20, DEBUGWARNING=15, IO=12, DEBUG=10` from the GUI, and the other values not exposed are `warning=30, error=40, critical=50`

Perhaps a comment like:
```suggestion
	# Don't set a config value outside of what can be set by the GUI
```